p2p/filters.py

```diff
@@ -1,5 +1,21 @@
+from builtins import str
 import re
 
+try:
+    unicode = unicode
```

Comment:
Are we ever using "unicode" as a function again after all the changes made to the code.

Reply:
I don't think so. That chunk of code is mainly to polyfill `bytes` for Python2.7. It might be overkill, but it covered all the bases

Reply:
I think `u'some string'` uses `unicode()` under the hood, so maybe best to leave that in?